stl/inc/yvals_core.h

```diff
@@ -799,7 +799,21 @@
 #define _DEPRECATE_STDEXT_HASH_UPPER_BOUND
 #endif // ^^^ warning disabled ^^^
 

```

Comment:
This feature should be listed, in sorted order, at the top of `yvals_core.h` in the `// _HAS_CXX20 directly controls:` section, since you have conditional code for it.